connector/fyle_integrations_platform_connector/apis/base.py

```diff
@@ -62,6 +62,17 @@ def get_all_generator(self):
 
         return self.connection.list_all(query_params)
 
+    def post(self, payload):
```

Comment:
Which API requires post?

Reply:
Currently Tax Api needed this but we will be using Bulk in next phase 1 migration